### PR TITLE
Editiable assign consultation workflow

### PR DESCRIPTION
**Pull request by taylorn01**

### CODE REVIEW

**Review by OwenGalvia — APPROVED** [review]
This works well but there is a cardinality error when editing digital objects. Unfortunately the same bug was missed when we added this to excavation licensing.
I'm happy to approve this and have both editable workflows at he same level of complete then add a ticket for the digital object bug.

Or if you prefer, don't merge until issue is resolved.

File: coral/coral/media/js/views/components/plugins/edit-workflow.js [hunk 1]
```diff
@@ -56,6 +56,95 @@ define([
...
+            componentData[tile.nodegroup] = {
+              value: JSON.stringify({
+                tileData: koMapping.toJSON(tile.data),
+                resourceInstanceId: tile.resourceinstance,
+                tileId: tile.tileid,
+                nodegroupId: tile.nodegroup
+              })
+            };
+
+        })
```
Comment (taylorn01) [inline coral/coral/media/js/views/components/plugins/edit-workflow.js:79]: @OwenGalvia Please run prettier on this file

File: coral/coral/media/js/views/components/plugins/edit-workflow.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ define([
     this.WORKFLOW_EDIT_MODE_LABEL = 'workflow-edit-mode';
     this.WORKFLOW_COMPONENT_ABSTRACTS_LABEL = 'workflow-component-abstracts';
     this.WORKFLOW_RECENTLY_EDITED_LABEL = 'workflow-recently-edited';
-
+    const self = this
```
Comment (taylorn01) [inline coral/coral/media/js/views/components/plugins/edit-workflow.js:13]: @OwenGalvia Let's try stick to "this" and not assign to self any more

File: coral/coral/media/js/views/components/plugins/edit-workflow.js (hunk 1 above)
Comment (OwenGalvia) [reply coral/coral/media/js/views/components/plugins/edit-workflow.js]: This is prettied? It's simply a new chunk of code

**Review by OwenGalvia — APPROVED** [review]

**Pull request merged**